wala-java.gradle

```diff
@@ -0,0 +1,106 @@
+apply plugin: 'eclipse'
+apply plugin: 'java-library'
+apply plugin: 'java-test-fixtures'
+apply plugin: 'maven-publish'
```

Comment:
Will this lead us to publish more artifacts than we did before?  I didn't think we included `maven-publish` for every project with Java source before.

Reply:
I think we were already including `maven-publish` for every subproject with Java source:

https://github.com/wala/WALA/blob/17064550328e82c68fb1a8464b3c26ad56c83538/build.gradle#L57-L61

Reply:
Ah yes, I was confused. `gradle-mvn-push.gradle` is only included in some places